docs/user/plagiarism-check.rst

```diff
@@ -0,0 +1,162 @@
+.. _plagiarism-check:
+
+Plagiarism checks
+=================
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Plagiarism Check Overview
+--------------------------
+
+Artemis allows tutors and exercise instructors to check assignments submissions from students for plagiarism.
```

Comment:
We do not have tutors in Artemis. → Teaching assistant or editor?

Reply:
We have tutors. Look here: 
![image](https://user-images.githubusercontent.com/10652693/178941858-f4d899b7-7043-4f39-8714-5448b32dadf6.png)